package.json

```diff
@@ -52,13 +55,11 @@
     "eslint-plugin-react-hooks": "4.6.0",
     "eslint-plugin-react-refresh": "0.4.3",
     "husky": "8.0.3",
-    "jsdom": "22.1.0",
+    "jsdom": "^24.0.0",
     "lodash.merge": "4.6.2",
     "prettier": "3.0.3",
     "styled-components": "6.0.7",
-    "typescript": "5.0.2",
-    "vite": "4.4.5",
-    "vite-tsconfig-paths": "4.2.0",
-    "vitest": "0.34.3"
+    "typescript": "^5.4.2",
+    "vitest": "^1.4.0"
```

Comment:
idem.

Please try to remove all dependencies that aren't used.

Reply:
https://github.com/defi-wonderland/web3-react-boilerplate/commit/9a9683fc0ce55ee0a3c687f7ac80783cade96b46